### ISSUE TITLE
Update README with tutorial and installation instructions

### ISSUE THREAD

**Issue by lopeetall** [anoma/vamp-ir#45]

**Comment by cwgoes** [comment]
Yeah, I think it would be nice to have the README focus on a simple tutorial, and link to the VampIR book (once online) for full documentation. Also, I think we should explain a bit how VampIR relates to Juvix, to Taiga to possibly other higher-level languages which might compile to it, and to individual backends / proof systems.

Also, it looks like the links to "Design rationale" and "spec" are broken. This is because the specs repository has changed layout, but I think we should just move these files into the VampIR repository (or perhaps the VampIR book) directly.

**Comment by cwgoes** [comment]
Oh - and the README should definitely show the benchmarks (especially the good ones :grinning:)
